src/validation.cpp

```diff
@@ -3639,7 +3639,7 @@ bool CVerifyDB::VerifyDB(const CChainParams& chainparams, CCoinsView *coinsview,
             }
         }
         // check level 3: check for inconsistencies during memory-only disconnect of tip blocks
-        if (nCheckLevel >= 3 && pindex == pindexState && (coins.DynamicMemoryUsage() + pcoinsTip->DynamicMemoryUsage()) <= nCoinCacheUsage) {
+        if (nCheckLevel >= 3 && pindex == pindexState && (coins.DynamicMemoryUsage() + DB_PEAK_USAGE_FACTOR * pcoinsTip->DynamicMemoryUsage()) <= nCoinCacheUsage) {
```

Comment:
The changes in the rollback test are never flushed, so there is no need for compensating for a peak?